csharp-basics/exercises/Collections/Exercise9/PhoneDirectory.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+
+namespace PhoneBook
+{
+	public class PhoneDirectory
+	{
+		private SortedDictionary<string, string> _data;
+		private int _dataCount;
+
+		public PhoneDirectory()
+		{
+			_data = new SortedDictionary<string, string>();
+			_dataCount = 0;
+		}
+
+		public bool Find(string name)
```

Comment:
ir jēga no šitās metodes? 
domāju ka mierīgi var lietot visur kur šo lieto _data.ContainsKey(name)
baigo atšķirību neredzu.